# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/webrtc/libwebrtcglue/MediaConduitInterface.h
# Commit: 7888e4be3d7e
# Full Hash: 7888e4be3d7e633739cf590e73b0a3af0ef3f9fc
# Author: Andreas Pehrson <apehrson@mozilla.com>
# Date: 2021-11-01 18:17:33
# Regressor Bug: 1654112
# File Overlap Count: 5
# Description:
#   Bug 1654112 - Fix remote-inbound-rtp.packetsReceived. r=bwc
#   
#   There are two problems with this stat:
#   - The BaseSeq tracking logic in MediaPipeline doesn't account for multiple ssrcs
#     (rtx)
# ==============================================================================

diff -r 6b3195b0e545 -r 7888e4be3d7e dom/media/webrtc/libwebrtcglue/MediaConduitInterface.h
--- a/dom/media/webrtc/libwebrtcglue/MediaConduitInterface.h	Mon Sep 06 16:04:25 2021 +0200
+++ b/dom/media/webrtc/libwebrtcglue/MediaConduitInterface.h	Wed Sep 08 10:34:57 2021 +0200
@@ -133,7 +133,10 @@
   virtual void ConnectSenderRtcpEvent(
       MediaEventSourceExc<MediaPacket>& aEvent) = 0;
 
+  // Sts thread only.
   virtual Maybe<DOMHighResTimeStamp> LastRtcpReceived() const = 0;
+  virtual Maybe<uint16_t> RtpSendBaseSeqFor(uint32_t aSsrc) const = 0;
+
   virtual DOMHighResTimeStamp GetNow() const = 0;
 
   virtual Ssrcs GetLocalSSRCs() const = 0;